### PR TITLE
Prettify

### PR DESCRIPTION
**Pull request by jdaudier**
Updated styling

@trezy Could you take what I've done and add your SCSS refactor on top of it? Sorry if I made it harder for ya.

**PREVIEW**
marionette js

### CODE REVIEW

**Comment by jdaudier** [comment]
I need help fixing this part... can't figure out why text won't wrap naturally.
wrap

**Comment by trezy** [comment]
@jdaudier No problem. This might make it even easier. ;-)

**Comment by jdaudier** [comment]
@trezy k, phew!

**Comment by samccone** [comment]
gosh that tuts+ screenshot is so bad

**Comment by jdaudier** [comment]
> gosh that tuts+ screenshot is so bad

i can fix that... give me a few min

File: src/sections/_books.jade [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+      img(src="images/books/requirejs.png", alt="Structuring Backbone Code with RequireJS and Marionette Modules")
+    a(href = "https://leanpub.com/marionette-serious-progression?utm_campaign=marionette-serious-progression&utm_medium=embed&utm_source=marionettejs.com", target= "_blank")
+      img(src="images/books/serious-progression.png", alt="Backbone.Marionette.js: A Serious Progression")
+    a(href = "http://www.amazon.com/Getting-Backbone-Marionette-Raymundo-Armendariz/dp/1783284250/ref=tmm_pap_title_0?ie=UTF8&qid=1420202176&sr=1-1", target= "_blank")
+      img(src="images/books/getting-started.png", alt="Getting Started with Backbone Marionette")
+    a(href = "https://leanpub.com/marionetteexpose?utm_campaign=marionetteexpose&utm_medium=embed&utm_source=marionettejs.com", target= "_blank")
+      img(src="images/books/expose.png", alt="Marionette Exposé")
+    a(href = "https://leanpub.com/building-backbone-plugins?utm_campaign=building-backbone-plugins&utm_medium=embed&utm_source=marionettejs.com", target= "_blank")
+      img(src="images/books/bb-plugins.png", alt="Building Backbone Plugins")
```
Comment (jasonLaster) [inline src/sections/_books.jade]: so much winn

**Pull request merged**